js/adapters/internal/Bitmovin8SpeedMeterAdapter.ts

```diff
@@ -0,0 +1,30 @@
+import {PlayerAPI} from 'bitmovin-player';
+import { DownloadSpeedMeter } from '../../core/DownloadSpeedMeter';
+
+export class Bitmovin8SpeedMeterAdapter {
+  private meter: DownloadSpeedMeter;
+
+  constructor(player: PlayerAPI, meter: DownloadSpeedMeter) {
+    this.meter = meter;
+    player.on(player.exports.PlayerEvent.DownloadFinished, this.handleOnDownloadFinishEvent)
```

Comment:
```suggestion
player.on(player.exports.PlayerEvent.DownloadFinished, (e) => this.handleOnDownloadFinishEvent(e));
```


If you don't use the arrow function, `this` will have a different scope in the `handleOnDownloadFinishedEvent`, and you won't be able to access `this.meter`